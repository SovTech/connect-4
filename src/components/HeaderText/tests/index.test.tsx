import * as React from 'react';
import * as renderer from 'react-test-renderer';
import { H1, H2, H3, H4 } from '../index';

describe('HeaderText Components', () => {
  it('renders a H1 correctly', () => {
    const tree = renderer
      .create(<H1 />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders a H2 correctly', () => {
    const tree = renderer
      .create(<H2 />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders a H3 correctly', () => {
    const tree = renderer
      .create(<H3 />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });

  it('renders a H4 correctly', () => {
    const tree = renderer
      .create(<H4 />)
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
