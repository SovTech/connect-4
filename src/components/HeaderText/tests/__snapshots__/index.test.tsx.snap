// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderText Components renders a H1 correctly 1`] = `
<h1
  className="sc-bdVaJa eBPzzR"
/>
`;

exports[`HeaderText Components renders a H2 correctly 1`] = `
<h2
  className="sc-bwzfXH ErJNR"
/>
`;

exports[`HeaderText Components renders a H3 correctly 1`] = `
<h3
  className="sc-htpNat gLVzZr"
/>
`;

exports[`HeaderText Components renders a H4 correctly 1`] = `
<h4
  className="sc-bxivhb kUrvWE"
/>
`;
